packages/block-editor/src/components/multi-selection-inspector/README.md

```diff
@@ -0,0 +1,28 @@
+# Multi Selection Inspector
+
+The `MultiSelectionInspector` component adds a card in the panel displayed by the [`BlockInspector`](https://github.com/WordPress/gutenberg/tree/master/packages/block-editor/src/components/block-inspector) component when several blocks are selected in the editor.
+
+This card contains information on the number of blocks selected, and in the case where one or more of these blocks contains text, the number of words they contain.
+
+![Multi selection inspector card](https://make.wordpress.org/core/files/2020/09/multi-selection-inspector-card.png)
+
+## Table of contents
+
+1. [Development guidelines](#development-guidelines)
+2. [Related components](#related-components)
+
+## Development guidelines
+
+### Usage
+
+Renders an block alignment toolbar with alignments options.
```

Comment:
I think this needs updating, probably from your template

Reply:
Oh yup. My bad.